modules/effects/arrow-effects/src/main/kotlin/arrow/effects/typeclasses/Bracket.kt

```diff
@@ -0,0 +1,83 @@
+package arrow.effects.typeclasses
+
+import arrow.Kind
+import arrow.core.Either
+import arrow.effects.typeclasses.ExitCase.Error
+import arrow.typeclasses.MonadError
+
+sealed class ExitCase<out E> {
```

Comment:
Should this be moved to `arrow.effects` like cats?

Reply:
[It comes along with `Bracket` into the same file in `cats-effect`](https://github.com/typelevel/cats-effect/blob/dbbbf3c30bd8b52d31bd95b3084aad1054cb483a/core/shared/src/main/scala/cats/effect/Bracket.scala#L146), so did the same here.